tika/src/test/java/fr/pilato/elasticsearch/crawler/fs/tika/TikaDocParserTest.java

```diff
@@ -667,6 +667,38 @@ public void testOcr() throws IOException, TikaConfigException {
         }
     }
 
+    @Test
+    public void testCustomTikaConfig() throws IOException, URISyntaxException {
+        InputStream tikaConfigIS = getClass().getResourceAsStream("/documents/tikaConfig.xml");
+        File tikaConfigFile = File.createTempFile("tikaConfigTestFile", ".xml");
```

Comment:
See the comments about integration tests. I think we should do something similar.